manifest/index.mjs

```diff
@@ -13,10 +37,9 @@ export function respondWithManifest(res, manifest){
    res.location(id)
    res.status(200)
    res.json(manifest)
-   res.end()
 }
 
-// Route performs the job
+// Expect an /{id} as part of the route, like /manifest/123
 router.route('/:id')
    .get(async (req, res, next) => {
       let id = req.params.id
```

Comment:
Project ID is most likely going to be a hexadecimal hash, so "is a number" is a bit sneaky.

Reply:
Fair.  Built it off of current functionality where the ID we get is an Integer Project ID.  Until a decision is made on exactly what it is any 'validation function' is just theoretical.  This is a stub to say "get ready to process a TPEN Project Entity Id here".  I added in the assumption an Integer number would suffice as the minimum, perfectly acceptable if not.  When we know what the ID is going to be, we can come in and update this validation.